### PR TITLE
Set values for the options

### PR DESCRIPTION
**Pull request by manst**
@squallstar 
When configuring a select, option or checkbox field add the ability to set a value not just a name

No ref to the GitHub issue

### CODE REVIEW

File: js/libs/core.js [hunk 1]
```diff
@@ -268,16 +268,31 @@ Fliplet.FormBuilder = (function() {
...
+            var regex = /<.*>$/g;
+            var match = rawOption.match(regex);
+            var option = {};
+
+            if (match) {
+              option.label = rawOption.replace(regex, '').trim();
+              var value = match[0].substring(1, match[0].length - 1).trim();
+              option.id = value || option.label;
+            } else {
+              option.label = option.id = rawOption;
```
Comment (squallstar) [inline js/libs/core.js]: split this into two separate assignments please

File: templates/components/select.build.hbs [hunk 2]
```diff
@@ -2,7 +2,7 @@
   <select class="form-control hidden-select" :name="name" :id="name" v-model="value" :required="required" v-on:change="updateValue()">
     <option v-if="placeholder" value="">\{{ placeholder }}</option>
     <option v-for="option in options" :value="option.id" :disabled="option.disabled">
-      \{{ option.id }}
+      \{{ option.label }}
```
Comment (squallstar) [inline templates/components/select.build.hbs]: what about existing forms that in their settings have only the `id`? We need to support those too for existing apps even if they don't go through the process of opening the form interface and saving changes. Does using a fallback work here? e.g. `{{ option.label || option.id }}`

Comment (manst) [reply templates/components/select.build.hbs]: Are there any cases how I can test this behavior with my changes?

File: js/libs/core.js (hunk 1 above)
Comment (manst) [reply js/libs/core.js]: Fixed

File: templates/components/select.build.hbs (hunk 2 above)
Comment (squallstar) [reply templates/components/select.build.hbs]: 1. setup a form locally before making your changes
2. switch the branch to include your changes
3. refresh the output iframe (http://localhost:3000/build) and see if works

You can also test out in a similar way on staging:

1. publish your form component without your changes
2. drop it into a screen and set it up as required
3. publish your form component with the changes (under the same package name)
4. refresh the screen (without making any change via the interface) and verify if it works

Comment (manst) [reply templates/components/select.build.hbs]: Fallback works. Have updated commit

**Pull request merged**